docs/node/run-your-node/validator-node.mdx

```diff
@@ -174,79 +174,56 @@ of that machine.
 To use this configuration, save it in the `/node/etc/config.yml` file:
 
 ```yaml title="/node/etc/config.yml"
-##
-# Oasis Node Configuration
-#
-# This file's contents are derived from the command line arguments found in the
-# root command of the oasis-node binary. For more information, execute:
-#
-#     oasis-node --help
-#
-# Settings on the command line that are separated by a dot all belong to the
-# same nested object. For example, "--foo.bar.baz hello" would translate to:
-#
-#     foo:
-#       bar:
-#         baz: hello
-##
-
-# Set this to where you wish to store node data. The node's artifacts
-# should also be located in this directory.
-datadir: /node/data
-
-# Logging.
-#
-# Per-module log levels are defined below. If you prefer just one unified log
-# level, you can use:
-#
-# log:
-#   level: debug
-log:
-  level:
-    # Per-module log levels. Longest prefix match will be taken. Fallback to
-    # "default", if no match.
-    default: debug
-    tendermint: warn
-    tendermint/context: error
-  format: JSON
-  # By default logs are output to stdout. If you would like to output logs to
-  # a file, you can use:
-  #
-  # file: /var/log/oasis-node.log
-
-# Genesis.
-genesis:
-  # Path to the genesis file for the current version of the network.
-  file: /node/etc/genesis.json
+mode: validator
+common:
+    # Set this to where you wish to store node data. The node's artifacts
+    # should also be located in this directory.
+    data_dir: /node/data
+    # Logging.
+    #
+    # Per-module log levels are defined below. If you prefer just one unified 
+    # log level, you can use:
+    #
+    # log:
+    #   level: debug
+    log:
+        format: JSON
+        level:
+            cometbft: warn
+            cometbft/context: error
+            # Per-module log levels. Longest prefix match will be taken. 
+            # Fallback to "default", if no match.
+            default: debug
+            # By default logs are output to stdout. If you would like to output 
+            # logs to a file, you can use:
+            #
+            # file: /var/log/oasis-node.log
```

Comment:
This part should be unindented and to make it more clear the keys should be ordered as in the old config, e.g. level, format, file.